Example/Backpack/View Controllers/BPKRootListTableViewController.m

```diff
@@ -1,57 +0,0 @@
-/*
- * Backpack - Skyscanner's Design System
- *
- * Copyright 2018-2020 Skyscanner Ltd
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#import "BPKRootListTableViewController.h"
-
-#import "Backpack_Native-Swift.h"
-#import <Backpack/Color.h>
-#import <Backpack/Icon.h>
-
-@interface BPKRootListTableViewController ()
-@property(weak, nonatomic) IBOutlet UIBarButtonItem *settingsButton;
-@end
-
-@implementation BPKRootListTableViewController
-
-- (void)viewDidLoad {
-    [super viewDidLoad];
-
-    if (ThemeHelpers.isThemingSupported) {
-        UIImage *largeSettingsIcon = [BPKIcon templateIconNamed:@"settings" size:BPKIconSizeLarge];
-        self.settingsButton.image = largeSettingsIcon;
-        self.settingsButton.accessibilityLabel = @"Settings";
-
-        self.settingsButton.target = self;
-        self.settingsButton.action = @selector(didTapSettingsButton);
-    }
-}
-
-- (void)didTapSettingsButton {
-    [ExampleApp showSettingsView];
-}
-
-- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
```

Comment:
Don't we still need this?

Reply:
I was thinking we could drop as we'll be migrating to using the new data, and it doesn't currently work anyway as it always says "Labels" in the app

Reply:
huh, fair enough then